CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# CHANGELOG
+
+\*\*
+
+Special thanks to external contributors on this release:
+
+## Release:
```

Comment:
@marbar3778 perhaps retro-fit / collect changes from the last two versions, as these are shown in the compatibility table: https://github.com/tendermint/rust-abci/tree/develop#tendermint-compatibility-table ?